File: Python/helicopter.py
from airway import Airway

class Helicopter(Airway):

    def __init__(self, vehicle_id, vehicle_name, engine_type, capacity):
        super().__init__(vehicle_id, vehicle_name, engine_type)
        self.capacity = capacity

    def display_details(self):
        super().display_details()
        print("Maximum Capacity:", self.capacity)

    def start_journey(self):
        print("Helicopter Journey Started Successfully...")

File: Java/Helicopter.java
class Helicopter extends Airway
{
    private int maxPassengers;

    Helicopter(int id, String name, String engineType, int maxPassengers)
    {
        super(id, name, engineType);
        this.maxPassengers = maxPassengers;
    }

    @Override
    public void displayDetails()
    {
        super.displayDetails();
        System.out.println("Maximum Capacity : " + maxPassengers);
    }

    @Override
    void startJourney()
    {
        System.out.println("Helicopter Journey Started Successfully...");
    }
}